Vagrantfile

```diff
@@ -0,0 +1,29 @@
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.network :private_network, ip: "192.168.33.10"
+
+  config.vm.synced_folder "./app", "/vagrant/app", id: "vagrant-root",
```

Comment:
Why not keep the default "." "/vagrant" mounting? Than we can remove this section.